kcmdline.c

```diff
@@ -1,15 +1,25 @@
 #include "kcmdline.h"
+#ifdef LINUX_TEST
+#include "linux_efilib.h"
+#else
+#include <efilib.h>
+#endif
 
-static const CHAR8 allowed[6][12] = {
+// If a provided command line has more tokens (words) than MAX_TOKENS
+// then an error will be returned.
+#define MAX_TOKENS 128
+
+// These are the tokens that are allowed to be passed on EFI cmdline.
+static const CHAR8 allowed[][32] = {
 	"^console=",
 	"^root=oci:",
 	"root=atomix",
 	"ro",
 	"quiet",
-	"verbose"
+	"verbose",
+    "crashkernel",
```

Comment:
tab/spaces.

But also should we just make this "crashkernel=256M"?  Else it needs to be "^crashkernel=" right?  Just "crashkernel" won't match "crashkernel=256M"?

Reply:
fixed. with crashkernel=256M.